components/faq/questions.js

```diff
@@ -0,0 +1,227 @@
+      answer:
+        'Operation Code, much like software, is built from anywhere with an internet connection,' +
+        " and is not based in one location. While we're headquartered in Portland, the entire" +
+        ' organization is decentralized, including the board of directors and the core team. This' +
+        " allows us to more effectively serve the entire military community, whether they're" +
+        " veterans or military spouses, whether they're OCONUS or in-country. We have chapters" +
+        ' all over the nation. Use Slack chat and join the closest town to you!',
+    },
+    {
+      question: 'Who does Operation Code serve?',
+      answer:
+        "Operation Code serves our nation's finest who've worn the uniform and their families" +
+        ' who are interested in coding and software development. Our programs are offered at no' +
+        ' cost to the military community, including veterans, transitioning service members, and' +
+        ' military spouses and families.',
+    },
+    {
+      question:
+        'What do I need to start learning software development and how much is this going to cost?',
+      answer:
+        'First off, you will need access to a solid computer, a good browser (ex. Chrome), and a' +
+        ' strong internet connection. It costs nothing to start learning to code and receive' +
+        ' software mentorship through Operation Code, you just need a positive attitude,' +
+        ' persistence and grit, and a thirst for new knowledge. We have over a dozen channels,' +
+        ' from Ruby to JavaScript, iOS to Android, and from Free Code Camp to edX study groups.',
+    },
+    {
+      question: 'What is available to start learning to code today?',
+      answer: (
+        <>
+          Our friends at the New York City-based, The Flatiron School created{' '}
+          <OutBoundLink href="https://learn.co/">Learn.co</OutBoundLink>, an online platform to get
+          introduced to web development and the popular web framework, Ruby on Rails. Request an
+          invite <OutBoundLink href="https://learn.co/join/operation-code">here</OutBoundLink> and
+          then join the #learn-dot-co channel in our Slack. Another resource is{' '}
+          <OutBoundLink href="https://www.learnhowtoprogram.com/courses">
+            learnhowtoprogram.com
+          </OutBoundLink>
+          , a resource maintained by Epicodus.
+        </>
+      ),
+    },
+    {
+      question: 'What are the hours of operation for Operation Code?',
+      answer: (
+        <>
+          Operation Code is different in that we don
+          {"'"}t have regular business office hours. The team can usually be found in our{' '}
+          <OutBoundLink href="http://operation-code.slack.com">Slack channel</OutBoundLink>, or on{' '}
+          <OutBoundLink href="https://github.com/OperationCode/operationcode">GitHub</OutBoundLink>{' '}
```

Comment:
Thanks so much for using the OutboundLink component. This helps us because the component has an out-of-the-box buy-in to our analytics. When people leave our site, we get to find out when/where and determine the why to make our site better.

For this system to work, we need to assign something to the `analyticsEventLabel` prop.

For all the QnA OutboundLink components can you put 
```
analyticsEventLabel="QnA Link"
```

Reply:
Ok cool, I'll get it added!